jwst/lib/exposure_types.py

```diff
@@ -25,6 +25,6 @@
 def is_moving_target(input_models):
     """ Determine if a moving target exposure."""
     model = input_models[0]
-    if model.meta.target.type == 'moving':
+    if model.meta.target.type.lower() == 'moving':
```

Comment:
@jdavies-st points out that many regtests are now failing because we have a lot of data files for non-moving target exposures that don't have the TARGTYPE keyword. So this logic should be changed slightly to first check for existence of model.meta.target.type and default to normal processing if it's not there.

Reply:
Will do. 
Is this a required keyword? In the future can we expect that it will be always populated (by SDP)?

Reply:
Yes, it's a required keyword in the KWD and SDP, and as such will always be populated with some string, where "UNKNOWN" is the default (in case SDP can't determine a value for it). So your logic is correct in that we should only ever apply mt processing if it's set to "MOVING". It's just some of our old test data that don't have it.

Reply:
Checking

```
if model.meta.target.type is not None and model.meta.target.type.lower() == 'moving'
```

does the trick.  There's another change I'm adding in to our miri image regtests based on the failures last night.  I can do both of these in a PR @nden.

It is just taking a very long time to run the tests from home.

Reply:
I saw your offer too late.